test/e2e/storage_test.go

```diff
@@ -70,6 +70,14 @@ func testStorage(t *testing.T) {
 			}
 		}()
 	}
+	t.Run("pod metrics", func(t *testing.T) {
```

Comment:
Would such a test be better suited as part of metrics_test.go? May it'd be best in the long run to split the other metrics tests out of the creation suite

Reply:
This is dependent on a workload, for the sake of not increasing the test runtime anymore I felt it best to put it in the storage test. Ideally in the future we use the same workload for both the storage test, and the network tests.